packages/providers/fal/image-to-video/fal_image_to_video/cli.py

```
@@ -13,8 +13,13 @@
 from pathlib import Path
 from typing import Optional
 
+from ai_content_pipeline.registry import ModelRegistry
+import ai_content_pipeline.registry_data  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)